### PR TITLE
Release 2.1.7-rc1

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens , can you review this? Thanks

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,428 @@
...
+    in ban and move constraints
+
+- Public API changes since Pacemaker-2.1.6
+  + libcib: cib_t now supports transactions via new cib_api_operations_t
+    methods, new cib_transaction value in enum cib_call_options, and new
+    cib_t transaction and user members
+  + libcib: cib_t now supports setting the ACL user for methods via new
+    cib_api_operations_t set_user() method
+  + libcib: deprecate cib_api_operations_t methods inputfd(), noop(), quit(),
+    set_op_callback(), and signon_raw(), and
```
Comment (clumens) [inline ChangeLog:55]: And what?

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,428 @@
...
+
+- Fixes since Pacemaker-2.1.6
+  + logging: restore ability to enable XML trace logs by file and function
+    (regression introduced in 2.1.6)
+  + scheduler: avoid double free with disabled recurring actions
+    (regression introduced in 2.1.5)
+  + tools: consider dampening argument when setting values with attrd_updater
+    (regression introduced in 2.1.5)
+  + tools: wait for reply from crm_node -R (regression introduced in 2.0.5)
+  + attrd: avoid race condition at writer election
```
Comment (clumens) [inline ChangeLog:22]: Since we're in the process of reverting this, it probably shouldn't be in the changelog.

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,428 @@
...
+  + libcrmcommon,libpe_rules,libpe_status: move enum pe_ordering, struct
+    pe_action_wrapper_s, struct pe_tag_s, struct pe_ticket_s, struct
+    resource_object_functions_s, enum node_type, enum pe_action_flags, enum
+    pe_discover_e, enum pe_find, enum pe_link_state, enum pe_obj_types, enum
+    pe_quorum_policy, enum pe_restart, struct pe_action_s, struct pe_node_s,
+    struct pe_node_shared_s, struct pe_resource_s, struct pe_working_set_s,
+    enum action_fail_response, enum action_tasks, enum pe_print_options, enum
+    rsc_recovery_type, enum rsc_role_e, and enum rsc_start_requirement to
+    libcrmcommon
+  + libpacemaker,libpe_rules,libpe_status: use pcmk_action_t instead of pe_action_t
```
Comment (clumens) [inline ChangeLog:346]: You'll want a comma at the end of this line

File: ChangeLog (hunk 1 above)
Comment (kgaillot) [reply ChangeLog:55]: ah, leftover from shuffling things a bit

File: ChangeLog (hunk 2 above)
Comment (kgaillot) [reply ChangeLog:22]: whoops, thought that was a different thing

File: ChangeLog (hunk 3 above)
Comment (kgaillot) [reply ChangeLog]: thanks

**Comment by kgaillot** [comment]
updated changelog commit per review

**Pull request merged**